test/c/concolic.t

```diff
@@ -2,11 +2,11 @@
   Assert failure
   Model:
     (model
-      (symbol_1 i32)
-      (symbol_2 i32)
-      (symbol_3 i32)
-      (symbol_4 i32)
+      (symbol_6 i32)
       (symbol_5 i32)
-      (symbol_6 i32))
+      (symbol_4 i32)
+      (symbol_3 i32)
+      (symbol_2 i32)
+      (symbol_1 i32))
```

Comment:
Do you know why they're now printed in reverse order?

Reply:
Oops they are being printed in reverse order of creation. Let me fix this